pallets/subtensor/src/lib.rs

```diff
@@ -609,7 +609,6 @@ pub mod pallet {
 		MaxAllowedUidsExceeded, // --- Thrown when number of accounts going to be registered exceed MaxAllowedUids for the network.
 		TooManyUids, // ---- Thrown when the caller attempts to set weights with more uids than allowed.
 		TxRateLimitExceeded, // --- Thrown when a transactor exceeds the rate limit for transactions.
-		RegistrationDisabled // --- Thrown when registration is disabled
```

Comment:
I think @Rubberbandits is working on a sudo registration disabling mechanism, let's keep this enum for now.

Reply:
you are right. I did fix it.